test/e2e/framework/metrics_util.go

```diff
@@ -652,37 +632,24 @@ func VerifyEtcdMetrics(c clientset.Interface) (*EtcdMetrics, error) {
 		return nil, err
 	}
 
-	backendCommitDurationHistogam := getNewHistogram()
-	snapshotSaveTotalDurationHistogram := getNewHistogram()
-	peerRoundTripTimeHistogram := getNewHistogram()
-	walFsyncDurationHistogram := getNewHistogram()
-	secondToMillisecondRatio := float64(time.Second / time.Millisecond)
+	result := EtcdMetrics{}
+	result.BackendCommitDuration = make(HistogramVec, 0)
+	result.SnapshotSaveTotalDuration = make(HistogramVec, 0)
+	result.PeerRoundTripTime = make(HistogramVec, 0)
+	result.WalFsyncDuration = make(HistogramVec, 0)
```

Comment:
Can you move this code block into a `NewEtcdMetrics()` function?

Reply:
done.